resources/features/refapp-3.x/07-logout/logout.feature

```diff
@@ -0,0 +1,11 @@
+Feature: User Logout
+
+  Background:
+    Given the user login to the Outpatient Clinic
+
+  Scenario Outline: user logout successfully from the application
+    When the user logout from the application
+    Then the user should be redirect to the login page
+    Examples:
+      | ability  |
+      | able     |
```

Comment:
There should be an example section for the feature files. Otherwise it produce an error. So I update the pr with this [5298820](https://github.com/openmrs/openmrs-test-3refapp/pull/11/commits/529882063cdb6848791524700d243f29e43195af) by using a example section for the test.